worldedit-core/src/main/java/com/fastasyncworldedit/core/beta/implementation/processors/ExtentBatchProcessorHolder.java

```diff
@@ -2,7 +2,7 @@
 
 import com.fastasyncworldedit.core.beta.IBatchProcessor;
 import com.fastasyncworldedit.core.configuration.Settings;
-import com.fastasyncworldedit.core.object.changeset.AbstractChangeSet;
+import com.fastasyncworldedit.core.history.changeset.change.AbstractChangeSet;
```

Comment:
AbstractChangeSet isn't a change per se. The change package should just be the for individual change-representing classes such as MutableBlockChange